### PR TITLE
Zeeshan in edit polls 59

### PR DESCRIPTION
**Pull request by sk-izsk**

### CODE REVIEW

File: server/routes/votes/votes.js [hunk 1]
```diff
@@ -10,48 +10,56 @@ router.post('/votes/:pollId', authentication, async (req, res) => {
...
-      await Poll.updateOne({ userId: pollOwnerId, _id: pollId }, { $pull: { votesForUrl2: userId } });
-      await Poll.updateOne({ userId: pollOwnerId, _id: pollId }, { $push: { [votesArray]: userId } });
-      const response = await Poll.find({ userId: pollOwnerId });
-      res.status(200).json(response);
-      return;
-    }
-    if (
-      (pollOwnerSpecificPoll.votesForUrl1.includes(userId) &&
+    // need to add the condition to find if the friend is invited in the poll or not
+    if (pollOwnerId !== userId && pollOwnerDetails.friends.includes(userId)) {
```
Comment (sk-izsk) [inline server/routes/votes/votes.js]: @SsaaaadD2  here you need to add the condition whether friend is invited or not

File: server/routes/users/users.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      }),
+    );
+    dupUserListVote1.map((elem) => (elem.votesFor = 'img1'));
+    dupUserListVote2.map((elem) =>
+      result.push({
+        name: elem.name,
+        avatar: elem.avatar ? elem.avatar : null,
+        votesFor: 'img2',
+      }),
+    );
```
Comment (aeciorc) [inline server/routes/users/users.js]: Why do you use a map when you don't store the result? It makes more sense to use a forEach here

Comment (sk-izsk) [reply server/routes/users/users.js]: fix the comment

File: client/src/components/PollFrame/PollFrame.jsx [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+    await deletePolls(_id).then((resp) => {
+      history.push('/home');
+    });
+  };
+
+  useEffect(() => {
+    getUserList({ votesForUrl1, votesForUrl2 }).then((result) => setUserList(result));
+  }, [votesForUrl1, votesForUrl2]);
+
+  console.log('this is votes', userList);
```
Comment (aeciorc) [inline client/src/components/PollFrame/PollFrame.jsx]: remove the log

File: client/src/pages/Login/Login.jsx [hunk 4]
```diff
@@ -56,6 +57,8 @@ const Login = () => {
     handler(event);
   };
 
+  console.log('this is context', action);
```
Comment (aeciorc) [inline client/src/pages/Login/Login.jsx]: rm

**Pull request merged**